administration/serial.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: Serial port configuration
+---
+
+{% include base.html %}
+
+# Serial port configuration
+
+Without correct configuration, binding using serial devices might not be able to open the serial port for communication, and you will see an error message in the logs.
+
+Following section gives serial configuration advice with different platforms.
+
+## Windows
+
+No special configuration is needed, you can use the COM port number (e.g. `COM4`) in the thing configuration.
+
+## Linux
+

```

Comment:
We should mention right at the start that openHABian actually comes with a menu option to configure the serial ports automatically. So all the rest here is only relevant for non-openHABian users.

Reply:
done